pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -63,6 +76,159 @@ type futureResponse struct {
 // Connection is the struct by which the sentry communicates with the FUSE server daemon.
 type Connection struct {
 	fd *DeviceFD
+
+	// Initialized after receiving FUSE_INIT reply.
+	// Until it's set, suspend sending FUSE requests.
+	Initialized bool
```

Comment:
Why Initialized is public, but initializedLock is private. We can't read Initialized without locking initializedLock.

Why are other attributes of this struct public?

Reply:
The lock was a temporary solution for the sync problem on `Initialized`. I will remove the lock and use `atomic` package.

About other attributes, they are designed to be used as the parameters through the whole lifetime of this connection. Actually the whole `Connection` struct is not supposed to be exported as well. So if that is a concern, we can un-export this struct?